torch/distributions/multinomial.py

```diff
@@ -40,6 +40,7 @@ class Multinomial(Distribution):
     """
     arg_constraints = {'probs': constraints.simplex,
                        'logits': constraints.real}
+    total_count: Integral
```

Comment:
Why this one can not be just `int`?
```suggestion
    total_count: int
```